lookout/core/metrics/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""Logging metrics for Prometheus."""
+from lookout.core.metrics.server import start_prometheus, submit_event
```

Comment:
`start_prometheus` should be lazy and hidden. The user should only know about `submit_event`, the rest is magic.

Reply:
Then I have no idea about how to do that magic without converting it in a pain in the ass or making it impossible to debug in case something fails.